pages/data/index.vue

```diff
@@ -668,27 +587,62 @@ export default {
       )
     },
 
-    /**
-     * Get filters based on the search type
-     */
-    fetchFilters: function() {
-      this.filters = []
-      this.isLoadingFilters = true
-
-      client
-        .getEntry(this.searchType.filterId, { include: 2 })
-        .then(response => {
-          const filters = transformFilters(response.fields)
-          this.filters = this.setActiveFilters(filters)
-        })
-        .catch(() => {
-          this.filters = []
-        })
-        .finally(() => {
-          this.isLoadingFilters = false
-        })
+    fetchFacets: function() {
```

Comment:
Would this be better off hard coded up the top or in a different file?

Reply:
moved it into data with a comment to remove once faceting logic is implemented